SQL/beestation_schema.sql

```diff
@@ -344,6 +344,7 @@ CREATE TABLE IF NOT EXISTS `SS13_poll_question` (
   `createdby_ckey` varchar(32) DEFAULT NULL,
   `createdby_ip` int(10) unsigned NOT NULL,
   `dontshow` tinyint(1) unsigned NOT NULL,
+  `minimumplaytime` int(4) NOT NULL,
```

Comment:
Missing CL entry. All DB changes need to be documented in the [database_changelog](https://github.com/BeeStation/BeeStation-Hornet/blob/master/SQL/database_changelog.txt)